pkg/controller/migmigration/backup.go

```diff
@@ -63,6 +68,55 @@ func (t *Task) ensureBackup() error {
 	return nil
 }
 
+// Ensure the second backup on the source cluster has been created and has the
+// proper settings.
+func (t *Task) ensureCopyBackup() error {
+	newBackup, err := t.buildBackup(nil)
+	if err != nil {
+		return err
+	}
+	uniqueBackupLabelKey := fmt.Sprintf("%s-%s", pvBackupLabelKey, t.PlanResources.MigPlan.UID)
```

Comment:
Why does the label reference the plan UID instead of the migration UID?

Reply:
No idea. Changed to reference the migration UID